### ISSUE TITLE
preventDefault doesn't work correctly

### ISSUE THREAD

**Issue by elcrostatina** [gorhom/react-native-animated-tabbar#45]
# Bug
the preventDefault method of the tabPress event doesn't work correctly, the focus remains on the tab that is pressed

## Steps To Reproduce

1. add the tabPress listener with the navigate method to the Tab.Screen
2. Run the App
3. Click to the relative tab
4. Press back icon to return to the previous screen

Describe what you expected to happen:
the focus have to remain on the previous tab

## Reproducible sample code

``<Tab.Screen
      name="Share"
      component={YourScreen}
      listeners={({navigation}) => ({
        tabPress: event => {
          event.preventDefault();
          navigation.navigate('YourScreen);
        },
      })}
    />``

![t_video5947238981388207483](https://user-images.githubusercontent.com/56482394/83191289-0dcafe80-a134-11ea-8e79-8249f875897f.gif)

**Comment by gorhom** [comment]
@elcrostatina thanks for reporting this issue, will look into it 👍

**Comment by Tyldar** [comment]
@gorhom Thanks for looking at this! Definitely need that feature 🙏 😄

**Comment by ThibaultJRD** [comment]
Any idea when the feature will be released? 😬
Thanks a lot 🙏🏼

**Comment by gorhom** [comment]
hi guys, sorry i am busy these days, but i got this fixed with this PR (https://github.com/gorhom/react-native-animated-tabbar/pull/50)

please test it out and let me know 👍
```bash
yarn add ssh://git@github.com:gorhom/react-native-animated-tabbar#feature/allow-parent-to-control-internal-index
```

**Comment by gorhom** [comment]
hi @elcrostatina , @ThibaultJRD , @Tyldar would mind test (#50) :)

```bash
yarn add ssh://git@github.com:gorhom/react-native-animated-tabbar#feature/allow-parent-to-control-internal-index
```

**Comment by elcrostatina** [comment]
> hi @elcrostatina , @ThibaultJRD , @Tyldar would mind test (#50) :)
> 
> ```shell
> yarn add ssh://git@github.com:gorhom/react-native-animated-tabbar#feature/allow-parent-to-control-internal-index
> ```

it works perfectly 💪💪

**Comment by gorhom** [comment]
this should be solved with [1.6.0](https://github.com/gorhom/react-native-animated-tabbar/releases/tag/v1.6.0)

**Comment by Tyldar** [comment]
Awesome thanks! It works perfectly :)
